functionalDB/functionalDB.markdown

```diff
@@ -1,4 +1,4 @@
-title: Designing a Database Like an Archaeologist
+title: An Archaeology-Inspired Database

```

Comment:
It was too long to render in chapter headers and Table of Contents, yes. I did change the second headline to be identical to your original title, so that your point is still made early in the chapter.